### DISCUSSION TITLE
Firefox YouTube video response

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#17] [Extra Credit]
### Worth 5 points

Watch the following YouTube opinion about [Firefox's role for a healthy web](https://www.youtube.com/watch?v=eA8O97U1Pbc) (15 minutes) and reflect on the following prompt. 

#### Warning: explicit words and more technical

> ## Do you agree or disagree that having multiple web browser engines is good for the internet? Why or why not?

Post your thinking cap about the video below.

**Comment by emmashanson** [comment]
Having multiple web browsers is in fact good for us for a few reasons. To begin with, having diversity and a balance of users allow for greater innovation and reasons in order to improve program developments that can supplement all of the various browsers. In the current state, Chromium makes up 85% of market shares, and is the primary web browser users have as their search engine. When there is a primary search engine by such a large margin, programmers will be less inclined to develop programs that support anything besides Chromium if few other users use other softwares. Furthermore, if a smaller search engine like Firefox decides to be innovative in an attempt to implement fresh, new changes, if Chromium has a pushback on that, developers will not have programs that support it because there will be no users for it. With this situation, a rabbit hole is created where there is a need for diversity and yet when smaller engines create new methods to be innovative and separate themselves from a larger search engine, developers are less inclined to program for an engine that also isn’t supported by chrome because of the financial difference in margin, and then the search engine just becomes smaller and less used by users.  A potential solution for this would include having a large pre-established company come out with a search engine that can attract a lot of users and from there, balance out the overall usage of the types of web browsers.

**Comment by penelopelamb** [comment]
Today, Chrome is the dominant web browser, consuming 85% of the market. It's true that the rise of Chromium is a result of a free market; Mozilla fell behind due to its tardiness to the mobile scene. However, a developer's priority is to create something that will work for the majority of people. If hardly anyone is using other engines like Firefox, developers are going to be less inclined to ensure compatibility with these browsers. As a result, when untested features don't work with Mozilla, the user is simply going to stick to what works (Chromium) and stray from what doesn't (Mozilla). So, as it stands, it's unlikely that Chromium's monopoly is going anywhere any time soon. The concern with the situation is the potential for abuse of power. Google defines the standard for what can and can't work on Chromium. The internet should not be restricted to any limits set by one single company, which is why it is important to incorporate the usage of multiple web browsers. Effectively, having multiple web browsers works towards distributing the market, enabling a more open internet for its users.

**Comment by antonyblin** [comment]
Having multiple web browser engines is absolutely good for the internet. In general, there are very few cases in which a monopoly in any industry is beneficial, and this is no different. At this point Chrome and Chrome/Chromium-based browsers dominate the market, and as a result, many web developers and designers often do not test for bugs and errors on non-Chrome/Chromium browsers, further leading to the exacerbation of the disuse of other browsers. Because of this, there is relatively little room for other browsers to create new features and capabilities for developers to take advantage of, since the majority are not bothering to test with these browsers in the first place. Additionally, Google owns a number of products that work exceedingly well in conjunction with Chrome-based browsers, but this is not so much the case for non-Chrome-browsers, as they have the ability to implement poorer quality versions of their products on these browsers, bringing even more users to Chrome/Chromium; one might liken this issue with the net neutrality cases that came into the mainstream over the last few years.

**Comment by ananyakodali17** [comment]
I agree that having multiple web browser engines is good for the internet. Currently, Google has an almost complete monopoly over web browser engines, and because of this web developers for the most part do not try to implement features that would not work on Chromium browsers or test for bugs and errors on non-Chrome browsers. This just further perpetuates the cycle of people choosing Chrome over everything because not all sites work on other browsers, though it is not the fault of the browser but rather the developers not testing on the browsers. Because of this, Google has a huge power over the internet and what can and cannot work, and because it is a corporation that has proved itself to not be trustworthy in the past, this is cause for some concern. There is no reason for people to switch to a non-Chromium browser, giving web developers no reason to utilize different features allowed on other search engines such as Firefox, therefore giving people no reason to switch browsers, creating a chicken-and-egg type scenario. This issue will likely not change due to the free-market nature of the situation, giving Google far too much power in the end and as they have proved to not be trustworthy with this power, having multiple web browser engines is better for the internet. Additionally, when there are multiple browsers in competition with each other, web developers are more likely to test on multiple platforms, each browser is more likely to continue to push to increase their standards to compete with the others, and no one browser has too much power over the others.

**Comment by kris10fu** [comment]
Having multiple web browser engines is good for the internet because it helps prevent one browser engine from having a monopoly. Currently, as mentioned in the video, Chromium has a monopoly over the browser engine market. This is a problem because very few developers will implement features that are not supported by the most mainstream browser, which in this case would be Chrome or Chromium-based browsers, as many users won’t be able to use them. This hinders the evolution of the internet as there is less incentive to implement innovative and original changes if the most-used browser will not support them. Smaller browser engines thus do not have a chance to shine and be known to more users. Additionally, another problem with the monopoly is that Google decides what goes in and what does not go into Chromium. This is concerning because that means a large corporation whose main purpose is to make money can mainly decide how the internet progresses in terms of web browser engines. Having web browser engine diversity via 2 or 3 engines with almost equal market share would be beneficial because, even if one of them does not implement new technology, it would not hold back the entire web. Instead, the other web engines would be able to implement new features. This may also push the engines to implement new changes if they see that the other engines are adding them as well. Thus, having multiple web browsers prevents the monopolization of the market, encouraging more equal distribution and innovation on the web.

**Comment by bittertang** [comment]
Having multiple web browser engines is necessary for the progression and evolution of the web. Currently, Chromium holds a monopoly in the web browser market, representing about 85% market shares. Despite being open source, Chromium is controlled by Google, and this can be dangerous because if potential new features interfere with Google’s own interests, such as tracking and ads, Google can choose not to implement them. Thus, there is little incentive for developers to create new features if the dominating web browser doesn’t allow them to work. Also, if a new browser attempts to enter the market without the resources of established corporations such as Google and Microsoft, it’d be very unlikely to succeed. That’s why it’s important to have engine diversity, where there are a variety of web browsers that have almost equal market share. In this case, developers are more inclined to take risks and implement features in the interest of the users. Overall, monopolies stunt the growth of the web and hinders users from maximizing their experience on the web.

**Comment by codingdianat** [comment]
Personally, I think that having multiple web browser engines is important to the internet. I fear that having one main web browser will allow one company to have monopoly over the web. As we discussed in class, data is really important to the financial success of modern day tech companies. Having one company manage web browsers allows them to have access to the data of anyone who searches the web. With just one web browser, it is hard to regulate what they do with our data because we will be so dependent on them. Some may say that Google has a monopoly on web browsers since Chrome is extremely popular.

**Comment by joonyloony** [comment]
I agree that having multiple web browser engines is good for the internet, because in terms of competition and innovation, I think that it is never good to have a monopoly. Currently, Chrome dominates the scene and even in my personal life, I can name a few complaints I have: if I don’t update the browser every week my computer is significantly slowed down, the extensions are not as functional as Firefox, and so on. But, since 2009 Firefox’s market share has been gradually declining and now only holds 4.2% of the market. Although it’s claimed that Chromium is open source, it was shown that about 92% of the edits to it were made by Google employees, so people are given a false sense of involvement and openness through this, and instead Google is slowly creating a monopoly over the market of internet browsers. Moreover, since there is little to no competition at this point, Google has no incentive to add helpful extensions and changes that would potentially upset their earnings potential (such as the given example of adding a great new technology that could potentially mess up the ad tracking). The Linux Experiment basically sums it up and says that the only way that having one dominant search engine is a good thing is if that engine is run by an “extremely community driven project and not by a big company that happened to open source the code and decides what goes in and not” and reaps revenue from the search engine.
